backend-tests/tests/infra/cli.py

```diff
@@ -18,8 +18,19 @@ class CliUseradm:
     def __init__(self):
         self.cid = docker.getid('mender-useradm')
 
+        # is it an open useradm, or useradm-enterprise?
```

Comment:
I would document that when both are available, the enterprise version is used (according to the code)